src/test/java/org/jetbrains/bsp/bazel/BazelBspServerTest.java

```diff
@@ -18,29 +19,82 @@
   private static final Integer SUCCESS_EXIT_CODE = 0;
   private static final Integer FAIL_EXIT_CODE = 1;
 
-  private final TestClient client;
   private final ExecutorService executorService = Executors.newCachedThreadPool();
 
   public BazelBspServerTest() {
     LOGGER.info("Creating TestClient...");
 
-    this.client =
+    LOGGER.info("Created TestClient");
```

Comment:
it looks like nothing happens between these two messages now

Reply:
Can I just remove them then?

Reply:
only one should be removed imo

Reply:
Since the creation of the test clients changed to the `run` method, I think this sort of logging should be done there. I've changed it up a bit. Let me know what you think